client/src/components/user/Logout.js

```diff
@@ -10,13 +11,11 @@ export function Logout() {
   }
 
   return (
-    <div>
-      <Button
-        className='fixed bottom-2 bg-red-500 bg-opacity-80 text-white hover:bg-red-300'
```

Comment:
looks good. i dunno about window, but on my laptop i think this hover is too light